tests/CodeBehind/UnitTests/BuildTests.cs

```diff
@@ -423,13 +450,19 @@ void AssertHaveCompilerError (bool haveError, string fileName, int line)
 			Assert.That (haveError, Is.True, $"Expected compiler error CS0266 in {fileName}({line},?)");
 		}
 
-		void RunTest (string testName, bool many, Action<string, bool, LocalBuilder> runner)
+		// runner parameters are:
+		//       string testName: name of the test being run - make it unique (log files etc are stored there)
+		//             bool many: whether the test is building more than 20 layouts to test parallel builds
+		//              bool dtb: whether the test is running a design time build (DTB)
+		//  LocalBuilder builder: the object that runs the actual build session
+		//
```

Comment:
Should you use the `///` summary comments here? Or do we care?

The method would then be nicer to consume from an IDE _if_ Intellisense is working.